### PR TITLE
add new token types

### PR DESCRIPTION
**Pull request by ayang-rkjs**

### CODE REVIEW

**Review by thebrianchen — CHANGES_REQUESTED** [review]
Thanks for plumbing this through! Can you also include a changelog entry in `CHANGELOG.md`?

File: src/api/nft-namespace.ts [hunk 1]
```diff
@@ -93,7 +93,7 @@ export class NftNamespace {
   getNftMetadata(
     contractAddress: string,
     tokenId: BigNumberish,
-    tokenType?: NftTokenType,
+    tokenType?: NftTokenType.ERC1155 | NftTokenType.ERC721,
```
[the remark this answers is not in the document]
Comment (ayang-rkjs) [reply src/api/nft-namespace.ts:96]: interesting, would this be a concern for batch as well?

[the remark this answers is not in the document]
Comment (ayang-rkjs) [reply src/api/nft-namespace.ts]: lmk what you think about this

File: src/types/types.ts [hunk 2]
```diff
@@ -396,9 +396,21 @@ export enum AssetTransfersOrder {
...
+  NOT_A_CONTRACT = 'NOT_A_CONTRACT',
   UNKNOWN = 'UNKNOWN'
 }
 
+/**
+ * NftTokenTypes that are allowed as request inputs.
+ *
+ * @public
+ */
+export type InputNftTokenType =
```
Comment (thebrianchen) [inline src/types/types.ts:409]: The `/types` directory is exported into the package namespace. We're only using `InputNftTokenType` as an internal type. Can you move this type into  `nft-api.ts` ?

**Review by thebrianchen — APPROVED** [review]
One minor fix, and looks good!

File: src/types/types.ts (hunk 2 above)
Comment (ayang-rkjs) [reply src/types/types.ts:409]: ouais

**Comment by Ivanrud77** [comment]
Thank you!

**Pull request merged**